File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        s=0
        temp=x
        while(x>0):
            r=x%10
            s=(s*10)+r
            x=x//10
        
        if(temp==s):
            return True
        else:
            return False

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int temp=x;
        int s=0,r=0;
        while(x>0){
            r=x%10;
            s=(s*10)+r;
            x=x/10;
        }
        if(temp==s){
            return true;
        }
        return false;
    }
}